sdk/message/src/sanitized.rs

```diff
@@ -37,15 +37,19 @@ pub struct LegacyMessage<'a> {
 }
 
 impl LegacyMessage<'_> {
-    pub fn new(message: legacy::Message, reserved_account_keys: &HashSet<Pubkey>) -> Self {
+    pub fn new(
```

Comment:
this crate follows semver so this will probably have to wait until SDK crates are moved to a new repo and given a new release schedule

Reply:
haven't heard anything about 3.0 coming soon. I think the SDK repo is the most imminent thing that will enable this

Reply:
Do we have a rough time line? Otherwise I will look for alternative implementations

Reply:
I think we discussed this offline last week, but for public visibility, the sdk will be moved into a new repo with the v2.2 branch cut, likely in the next 7-10 days.

After that point, we can consider making breaking changes to `solana-message` or any other crate. We need to choose what to do with `solana-program` and `solana-sdk` -- my inclination is to bump major versions much less often for them, or stop publishing updates altogether and encourage people to use the split-up crates.